cameraServer.py

```diff
@@ -0,0 +1,215 @@
+        str = j["ntmode"]
+        if str.lower() == "client":
+            server = False
+        elif str.lower() == "server":
+            server = True
+        else:
+            parseError("could not understand ntmode value '{}'".format(str))
+
+    # cameras
+    try:
+        cameras = j["cameras"]
+    except KeyError:
+        parseError("could not read cameras")
+        return False
+    for camera in cameras:
+        if not readCameraConfig(camera):
+            return False
+
+    return True
+
+"""Start running the camera."""
+def startCamera(config):
+    print("Starting camera '{}' on {}".format(config.name, config.path))
+    cs = CameraServer.getInstance()
+    camera = cs.startAutomaticCapture(name=config.name, path=config.path)
+    camera.setConfigJson(json.dumps(config.config))
+    return cs, camera
+
+
+
+#Process Functions
+
+def getRetroPos(img, display=False, sample=False):
+    """Function for finding retro-reflective tape a"""
+    hsv = cv2.cvtColor(img, cv2.COLOR_BGR2HSV) #Convert to HSV to make the mask easier
+    if sample:
+        mask = cv2.inRange(hsv, sample_lowerGreen, sample_higherGreen) #Create a mask of everything in between the greens
+    else:
+        mask = cv2.inRange(hsv, lowerGreen, higherGreen)
+    mask = cv2.dilate(mask, None, iterations=2) #Expand the mask to allow for further away tape
+
+    contours = cv2.findContours(mask, 1, 2)[-2] #Find the contours
+    
+    if len(contours) > 1: #Get contours with area above magic number 10 and append its smallest rectangle
+        rects = []
+        for cnt in contours:
+            if cv2.contourArea(cnt) > minContourArea:
+                rects.append(cv2.minAreaRect(cnt))
+        rects.sort(key=lambda x:x[2])
+        
+        pairs = []
+        leftRect = None
+        for i, rect in enumerate(sorted(rects, key=lambda x:x[0])): #Get rectangle pairs
+            if rect[2] < (leftAngleSize + angleOffset) and rect[2] > (leftAngleSize - angleOffset):
+                leftRect = rect
+            elif rect[2] < (rightAngleSize + angleOffset) and rect[2] > (rightAngleSize - angleOffset):
+                if leftRect:
+                    pairs.append((leftRect, rect))
+                    leftRect = None
+                    
+        if len(pairs) >= 1:
+            closestToMiddle = min(pairs, key = lambda x:abs(x[0][0][0] - screenSize[0]/2))
+        else:
+            return None, None, img
+
+        circleContours = list(np.int0(cv2.boxPoints(closestToMiddle[0])))
+        circleContours.extend(list(np.int0(cv2.boxPoints(closestToMiddle[1]))))
+        circleContours = np.array(circleContours)
+        (x,y),radius = cv2.minEnclosingCircle(circleContours)
+        radius = int(radius)
+        center = (int(x), int(y))
+
+        angle = (closestToMiddle[0][1][0] * closestToMiddle[0][1][1]) / (closestToMiddle[1][1][0] * closestToMiddle[1][1][1])
+        
+        if display == True: #Create the annotated display if display is True
```

Comment:
Save character use [truth value testing](https://docs.python.org/3/library/stdtypes.html#truth-value-testing)
```suggestion
        if display: #Create the annotated display if display is True
```